packages/nimble-components/src/tabs/specs/scrollable-tabs.md

```diff
@@ -0,0 +1,106 @@
+# Scrollable tabs
+
+## Problem Statement
+
+There are some use-cases that call for many tabs, and in cases where the width of all of the tabs exceed the screen space, they will bleed off screen (and potentially make the page scrollable to see the other tabs). To prevent this, we will introduce "scrollable" tabs, which will enforce the tabs to be constrained to the given space of the `Tabs` component and offering buttons to allow a user to navigate through the whole set of tabs.
+
+### Out of scope of this HLD
+
+Any decision/API around limiting the width of a particular tab. This is ultimately an orthogonal concern to the UX this HLD is presenting.
+
+## Links To Relevant Work Items and Reference Material
+
+-   [Issue 1509: Scrollable Tabs](https://github.com/ni/nimble/issues/1509)
+-   [Figma design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=11133-57220&node-type=instance&t=DlnrdRo7KWYlheFz-0)
+
+## Implementation / Design
+
+![Scrollable Tabs](./spec-images/scrollable-tabs.png)
+
+The following decisions were agreed to in the comments section in the linked issue above:
```

Comment:
Unless it's complicated, I think we should support scrolling the tabs via mouse scroll wheel (when cursor is over a tab).